src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<www-page>
+		<kv-page-container>
+			<!-- Page Content -->
+		</kv-page-container>
+	</www-page>
+</template>
+
+<script>
+import logReadQueryError from '@/util/logReadQueryError';
+import WwwPage from '@/components/WwwFrame/WwwPage';
+import lenderPublicProfileQuery from '@/graphql/query/lenderPublicProfile.graphql';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+
+export default {
+	name: 'LenderProfile',
+	inject: ['apollo', 'cookieStore'],
+	components: {
+		WwwPage,
+		KvPageContainer,
+	},
+	metaInfo() {
+		return {
+			title: this.pageTitle,
+			meta: [
+				{ property: 'og:title', vmid: 'og:title', content: this.pageTitle },
+				{ property: 'og:description', vmid: 'og:description', content: this.pageDescription },
+				{ property: 'og:site_name', vmid: 'og:site_name', content: 'Kiva' },
+				{
+					vmid: 'description',
+					name: 'description',
+					content: this.pageDescription,
+				}
+			].concat([
+				{
+					vmid: 'facebook_label',
+					name: 'facebook_label',
+					content: `Kiva - Lender > ${this.lenderName} from ${this.lenderWhereAbouts}`
+				},
+			]).concat([
+				{
+					vmid: 'robots',
+					name: 'robots',
+					content: 'noindex, nofollow',
```

Comment:
Thanks for adding this! This will be good to have while we work on this page. We will have to remember to remove it once this page is ready for production.

Reply:
@roger-in-kiva @emuvente do we have a follow-up ticket?

Reply:
@dyersituations  I added a comment here: https://kiva.atlassian.net/browse/MP-486